frontend/src/pages/search/styles.scss

```diff
@@ -0,0 +1,133 @@
+@import "~/styles/variables.scss";
+@import "~/styles/layouts.scss";
+
+.page {
+  @include container;
+}
+
+.header {
+  font-size: 2em;
+  .search-term {
+    color: $purple;
+    font-weight: 600;
+  }
+}
+
+.listing-top-bar {
+  display: flex;
+}
+
+.listing {
+  display: flex;
+  flex-direction: column;
+
+  .listing-item {
+    padding: 12px 0;
+    border-bottom: 1px solid $grey-2;
+
+    display: grid;
+    grid-template-columns: repeat(auto-fill, 1fr);
+    grid-template-rows: repeat(4, auto-fill);
+    grid-template-areas:
+      "title-row     downloads-counter"
+      "description   downloads-counter"
+      "tag-container downloads-counter"
+      "bottom-row    downloads-counter";
+    grid-gap: 5px;
+
+    .listing-item__title-row {
+      grid-area: title-row;
+
+      display: flex;
+      align-items: flex-end;
+
+      *:not(:first-child) {
+        margin-left: 5px;
+      }
+
+      .listing-item__title {
+        font-size: 1.3rem;
+        font-weight: bold;
+        font-family: $family-secondary;
+      }
+
+      .listing-item__version {
+        font-size: 1rem;
+        font-family: $family-code;
+        color: $grey-5;
```

Comment:
(used in many places, but just leaving a comment here): grey-5 is a bit hard to read on white background and doesn't meet WCAG AA contrast guidelines, which isn't much of a problem when you have good vision and a decent monitor, but it's enough to count one of those out for it to be an issue.
As a tip, there's a plugin called "WCAG AA Contrast Checker" which makes spotting this stuff easier.